src/commodore/index.ts

```diff
@@ -38,7 +38,7 @@ export const defaultExtraConfig = {
   distributionRegex:
     /^distribution\/(?<distribution>[^\/]+)(?:\/cloud\/(?<cloud>.+)\.ya?ml|(?:\/.+)?\.ya?ml)$/,
   cloudRegionRegex:
-    /^cloud\/(?<cloud>[^\/]+)(?:\/(?<region>.+)\.ya?ml|\.ya?ml)$/,
+    /^cloud\/(?<cloud>[^\/]+)(?:\/(?<region>[^\/]+)\.ya?ml|.*\.ya?ml)$/,
```

Comment:
I'd prefer to not match on `cloud/<cloudname>/.yaml`, but it probably doesn't make a large difference.

```suggestion
    /^cloud\/(?<cloud>[^\/]+)(?:\/(?<region>[^\/]+)\.ya?ml|.+\.ya?ml)$/,
```

Reply:
That doesn't match `cloud/foo.yaml`. The following should work though `/^cloud\/(?<cloud>[^\/]+)(?:\/(?<region>[^\/]+)\.ya?ml|[^\/]*\.ya?ml)$/`